src/structures/paging/mapper/mod.rs

```diff
@@ -55,20 +61,26 @@ pub enum TranslateResult {
         frame: PhysFrame<Size4KiB>,
         /// The offset whithin the mapped frame.
         offset: u64,
+        /// Flags in the pagetable for this entry
```

Comment:
We should clarify here that these are only the flags of the last-level page table entry. (Flags of higher-level page table entries can affect the effective flags for an address too, for example when the `WRITABLE` flag is not set for a level 3 entry.)